### PR TITLE
ticdc: update v4.0.3 changes

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed, added or deleted? (Required)

Update v4.0.3 changes.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### CODE REVIEW

**Comment by overvenus** [comment]
PTAL @amyangfei @TomShawn

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -95,11 +95,14 @@ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --addr=0.0.0.0:830
...
+cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:123456@127.0.0.1:3306/" --changefeed-id="simple-replication-task"
+Create changefeed successfully!
+ID: simple-replication-task
+Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":"2020-03-12T22:04:08.103600025+08:00","start-ts":415241823337054209,"target-ts":0,"admin-job-type":0,"sort-engine":"memory","sort-dir":".","config":{"case-sensitive":true,"filter":{"rules":["*.*"],"ignore-txn-start-ts":null,"ddl-allow-list":null},"mounter":{"worker-num":16},"sink":{"dispatchers":null,"protocol":"default"},"cyclic-replication":{"enable":false,"replica-id":0,"filter-replica-ids":null,"id-buckets":0,"sync-ddl":false},"scheduler":{"type":"table-number","polling-time":-1}},"state":"normal","history":null,"error":null}
 ```
 
-其中 `--sink-uri` 需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`。
+- `--changefeed-id`: 同步任务的 id，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。
```
Comment (amyangfei) [inline ticdc/manage-ticdc.md:104]: 可以说明一下是可选项，不指定会由系统自动生成

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -547,7 +560,7 @@ sync-ddl = true
 ### 环形同步使用限制
 
 1. 在创建环形同步任务前，必须使用 `cdc cli changefeed cyclic create-marktables` 创建环形复制功能使用到的标记表。
-2. 开启环形复制的数据表只包含 [a-zA-z0-9_] 字符。
+2. 开启环形复制的数据表名字需要符合正则表达式 `^[a-zA-Z0-9_]+$`。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:563]:
```suggestion
2. 开启环形同步的数据表名字需要符合正则表达式 `^[a-zA-Z0-9_]+$`。
```

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (overvenus) [inline ticdc/manage-ticdc.md:104]:
```suggestion
- `--changefeed-id`: 同步任务的 id，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`，如果不指定 TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -95,11 +98,14 @@ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --addr=0.0.0.0:830
 {{< copyable "shell-regular" >}}
 
 ```shell
-cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:123456@127.0.0.1:3306/"
-create changefeed ID: 28c43ffc-2316-4f4f-a70b-d1a7c59ba79f info {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":"2020-03-12T22:04:08.103600025+08:00","start-ts":415241823337054209,"target-ts":0,"admin-job-type":0,"config":{"filter-case-sensitive":false,"filter-rules":null,"ignore-txn-start-ts":null}}
+cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:123456@127.0.0.1:3306/" --changefeed-id="simple-replication-task"
```
Comment (amyangfei) [inline ticdc/manage-ticdc.md:101]: 创建任务可以增加一下使用 tls 相关的示例

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -159,6 +163,9 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
     | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `64MB`） |
     | `replication-factor` | kafka 消息保存副本数（可选，默认值 `1`）                       |
     | `protocol` | 输出到 kafka 消息协议，可选值有 `default`, `canal`（默认值为 `default`）    |
+    | `ca`       | 连接下游 KafKa 实例所需的 CA 证书文件路径（可选） |
+    | `cert`     | 连接下游 KafKa 实例所需的证书文件路径（可选） |
+    | `key`      | 连接下游 KafKa 实例所需的证书密钥文件路径（可选） |
```
Comment (overvenus) [inline ticdc/manage-ticdc.md:168]:
```suggestion
    | `ca`       | 连接下游 Kafka 实例所需的 CA 证书文件路径（可选） |
    | `cert`     | 连接下游 Kafka 实例所需的证书文件路径（可选） |
    | `key`      | 连接下游 Kafka 实例所需的证书密钥文件路径（可选） |
```

**Comment by overvenus** [comment]
PTAL, thanks!

File: enable-tls-between-components.md [hunk 5]
```diff
@@ -66,6 +66,16 @@ aliases: ['/docs-cn/dev/how-to/secure/enable-tls-between-components/']
         key-path = "/path/to/pd-server-key.pem"
         ```
 
+    - TiCDC
+
+        在启动命令行中设置，并设置相应的 URL 为 https：
```
Comment (TomShawn) [inline enable-tls-between-components.md:71]:
```suggestion
        在启动命令行中设置，并设置相应的 URL 为 `https`：
```

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -94,11 +101,14 @@ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --addr=0.0.0.0:830
...
+cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:123456@127.0.0.1:3306/" --changefeed-id="simple-replication-task"
+Create changefeed successfully!
+ID: simple-replication-task
+Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":"2020-03-12T22:04:08.103600025+08:00","start-ts":415241823337054209,"target-ts":0,"admin-job-type":0,"sort-engine":"memory","sort-dir":".","config":{"case-sensitive":true,"filter":{"rules":["*.*"],"ignore-txn-start-ts":null,"ddl-allow-list":null},"mounter":{"worker-num":16},"sink":{"dispatchers":null,"protocol":"default"},"cyclic-replication":{"enable":false,"replica-id":0,"filter-replica-ids":null,"id-buckets":0,"sync-ddl":false},"scheduler":{"type":"table-number","polling-time":-1}},"state":"normal","history":null,"error":null}
 ```
 
-其中 `--sink-uri` 需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`。
+- `--changefeed-id`: 同步任务的 id，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`，如果不指定 TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:110]:
```suggestion
- `--changefeed-id`: 同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
```

**Pull request merged**